### PR TITLE
[PHP-FPM] Add Integration Package with Pool Data Stream

### PR DESCRIPTION
**Pull request by harnish-crest**
- Enhancement

## What does this PR do?

- Added 1 data stream (Pool metrics).
- Added data collection logic for the data streams.
- Added the ingest pipeline for the data streams.
- Mapped fields according to the ECS schema and added Fields metadata in the appropriate yaml files.
- Added dashboards and visualizations.
- Added system test cases for the data stream.

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's changelog.yml file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

- Clone integrations repo.
- Install elastic-package locally.
- Start elastic stack using elastic-package.
- Move to (integrations/packages/php_fpm) directory.
- Run the following command to run tests.
elastic-package test

## Related issues
 
- Relates #636
- Relates #4652 

## Screenshots
php_fpm-pool-dashboard

### CODE REVIEW

**Review by kush-elastic — APPROVED** [review]

**Comment by kush-elastic** [comment]
LGTM

File: packages/php_fpm/_dev/deploy/docker/files/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM bitnami/php-fpm:8.1.12-debian-11-r4
```
Comment (ManojS-shetty) [inline packages/php_fpm/_dev/deploy/docker/files/Dockerfile:1]: Q:Are we not keeping this docker image in elastic repo? In here: https://container-library.elastic.co/

Comment (harnish-crest) [reply packages/php_fpm/_dev/deploy/docker/files/Dockerfile:1]: We never came across adding docker image in [elastic repo](https://container-library.elastic.co/) till now. Also we do not have the access for the elastic repo.

**Review by ManojS-shetty — APPROVED** [review]
LGTM!

**Pull request merged**